assets/app/views/modals/delete-resource.html

```diff
@@ -4,6 +4,21 @@
       '<strong>{{displayName ? displayName : resourceName}}</strong>'?</h1>
     <p>This<span ng-if="isProject"> will <strong>delete all resources</strong> associated with the project {{displayName ? displayName : resourceName}} and</span> <strong>cannot be undone</strong>.  Make sure this is something you really want to do!</p>
 
+    <div ng-show="isProject">
```

Comment:
instead of isProject, what if we make the extra confirmation an option on the directive? in case we want to do this with any other resources later